scripts/clone.cjs

```
@@ -0,0 +1,47 @@
+const { loadEnvConfig } = require("@next/env");
+const git = require("isomorphic-git")
+const http = require("isomorphic-git/http/node")
+const fs = require("fs")
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)